test/extensions/quic_listeners/quiche/active_quic_listener_test.cc

```diff
@@ -111,15 +111,18 @@ class ActiveQuicListenerTest : public QuicMultiVersionTest {
     ON_CALL(listener_config_, listenSocketFactory()).WillByDefault(ReturnRef(socket_factory_));
     ON_CALL(socket_factory_, getListenSocket()).WillByDefault(Return(listen_socket_));
 
-    // Use UdpGsoBatchWriter to perform non-batched writes for the purpose of this test
+    // UdpGsoBatchWriter cannot be supported on Windows, use UdpDefaultWriter instead
+    // TODO(ggreenway,wrowe): Permit batched UdpGsoBatchWriter when available in a platform
+    // agnostic manner leveraging PR #13086
+    // See discussion: https://github.com/envoyproxy/envoy/pull/12898#discussion_r492085735
     ON_CALL(listener_config_, udpPacketWriterFactory())
         .WillByDefault(Return(
             std::reference_wrapper<Network::UdpPacketWriterFactory>(udp_packet_writer_factory_)));
     ON_CALL(udp_packet_writer_factory_, createUdpPacketWriter(_, _))
-        .WillByDefault(Invoke(
-            [&](Network::IoHandle& io_handle, Stats::Scope& scope) -> Network::UdpPacketWriterPtr {
+        .WillByDefault(
+            Invoke([&](Network::IoHandle& io_handle, Stats::Scope&) -> Network::UdpPacketWriterPtr {
               Network::UdpPacketWriterPtr udp_packet_writer =
-                  std::make_unique<Quic::UdpGsoBatchWriter>(io_handle, scope);
+                  std::make_unique<Network::UdpDefaultWriter>(io_handle);
```

Comment:
The need for this change will go away after #13086 merges. 

@danzh2010 what was the reason for using the GSO batch writer in this test instead of the default writer? Coverage?

Reply:
Let's wait for the other PR to merge and then come back to this. It should merge soon.

/wait

Reply:
> I have an alternate suggestion for all the pragmas to disable warnings: Could we create quiche_include_pre.h and quiche_include_post.h, and bracket those around quiche includes, and put all the warning-disable flags, and compiler ifdefs for warning-disable flags, into the pre and post, so we don't duplicate all that junk in every file?

That's entirely feasible. In the long run, and with the desire to incorporate precompiled headers at some point, it makes sense for all includes to follow the same pattern which is precomputable in sequence. But I wouldn't have that hold up this patch, we are trying to arrive at a stable (if not quite ready) buildable windows binary by 1.16, so that extra complexity is outside the scope of this PR.

Reply:
(Sorry for the mislocated reply-to-thread :)